src/core/BufferAttribute.js

```diff
@@ -32,6 +32,12 @@ function BufferAttribute( array, itemSize, normalized ) {
 
 }
 
+Object.defineProperty( BufferAttribute.prototype, "needsUpdate", {
+
+	set: function(value) { if ( value === true ) this.version ++; }
+
+});
```

Comment:
Friendly tip: before you do too many of these, be aware of your formatting. :)
```javascript
Object.defineProperty( BufferAttribute.prototype, "needsUpdate", {

	set: function ( value ) {

		if ( value === true ) this.version ++;

	}

} );
```
http://zz85.github.io/mrdoobapproves/